### DISCUSSION TITLE
How can this be made to work longer than the 1 month Tuya API trial?

### DISCUSSION THREAD

**Discussion by von32** [prasad-edlabadka/homebridge-tuya-ir#94] [Q&A]
Hey there! First, thanks a million for creating this. I have an issue where my IoT Core API access expired- looks like it only lasts a month. Did anyone get this to work indefinitely? Support said I can use their api for free- but again, contract expired after a month, and I only get one trial. Did I do something wrong? the full license is 25,000 USD (lol).

**Comment by Nobleash** [comment]
You just need to renew the core API for 6 months every 6 months. Just write in the desciption its for a personal home integration and they auto approve it
